content/contributing/project-information/release-process.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Release Process"
+insert_anchor_links = "right"
+[extra]
+weight = 3
++++
+
+Bevy uses three-months-long development cycles, delimited by a weeks-long rolling release process.
+
+This means that bevy releases a new version roughly once every three to four months. In an effort to avoid crunch and encourage sustainable development, we do not pick specific dates or deadlines.
+
+## Development Cycles
+
+During a three-month development cycle, maintainers collect important PRs and issues into a GitHub Milestone. Issues included in the milestone are given priority, and the community generally makes a best-effort attempt to merge as many of these as possible over the duration of the cycle.
+
+In the lead-up to a release (generally about about two and a half months after the previous) we begin to aggressively purge the milestone of things that seem unlikely to be ready in time. The main things that tend to remain are:
+
+- Quick and easy features under active development.
+- Fixes for important regressions or crashes.
+- Large or impressive features that only need minor polish.
+
+When the milestone is complete, we publish a release candidate (ex. `0.14.0-rc.1`) and begin the release process.
+
+## Release Candidates
+
+In the run-up to a release, Bevy publishes a new release candidate about once a week. Publishing release candidates allows users to identify bugs and regressions, gives ecosystem crates time to update so they can be ready closer to release day, and provides bevy contributors extra time to write release notes and migration guides.
+
+When publishing a release candidate, we also announce a general target release date. This is generally about one week out. If critical bugs or regressions are discovered during this testing window, we fix them and publish a new release candidate. Then we either
+
+- Announce a new date, if we believe more testing time is needed than the current window, or
+- Keep the same target release date if we believe there is enough time in the current window.
+
+When we successfully hit a release candidate's target date, we do the final release.
+
+## Release Checklist
+
+When making a release, the maintainers follow these checklists:
+
+### Minor Version
+
+#### Minor Pre-release
+
+1. Check regressions tag.
+2. Check appropriate milestone and close it.
+3. Check GitHub Projects page for staleness.
+4. Update change log.
+5. Create migration guide.
+6. Write blog post.
+7. Update book.
+8. Bump version number for all crates, using the "Release" workflow.
+   1. Change the commit message to be nicer.
+9. Create tag on GitHub.
+10. Edit GitHub Release. Add links to the `Release announcement` and `Migration Guide`.
+11. Bump `latest` tag to most recent release.
+12. Run [the `update-screenshots` workflow] to update screenshots. _This will block blog post releases (and take ~40 minutes) so do it early_.
```

Comment:
```suggestion
12. Run [the `update-screenshots` workflow] to update screenshots. *This will block blog post releases (and take ~40 minutes) so do it early*.
```
Other documents use asterisks for italics; we should maintain that consistency.